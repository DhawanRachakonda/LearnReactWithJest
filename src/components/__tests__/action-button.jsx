import React from 'react';
import renderer from 'react-test-renderer';

import ActionButton from '../action-button.jsx';

it('Display Actionbutton', () => {
  const tree = renderer.create(
    <ActionButton className="button" text="Add" onAction={jest.fn()} />,
  );
  expect(tree.toJSON()).toMatchSnapshot();
});
