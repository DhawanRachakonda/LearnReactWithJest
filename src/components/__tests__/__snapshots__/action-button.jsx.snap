// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Actionbutton 1`] = `
<button
  className="action-button button"
  onClick={[MockFunction]}
  type="submit"
>
  <span>
    ADD
  </span>
</button>
`;
